app/src/main/java/org/oppia/android/app/profile/PinPasswordActivity.kt

```diff
@@ -51,6 +58,17 @@ class PinPasswordActivity :
     pinPasswordActivityPresenter.handleRouteToSuccessDialog()
   }
 
+  override fun onSaveInstanceState(outState: Bundle) {
+    super.onSaveInstanceState(outState)
+    val args = PinPasswordActivityStateBundle.newBuilder()
+      .setInputPin(pinPasswordActivityPresenter.getInputPin())
+      .build()
+    outState.putProto(
+      PINPASSWORD_ACTIVITY_STATE_KEY,
+      args
+    )
```

Comment:
Could this be a single line?

```suggestion
    outState.putProto(PINPASSWORD_ACTIVITY_STATE_KEY,args)
```

Reply:
Updated.